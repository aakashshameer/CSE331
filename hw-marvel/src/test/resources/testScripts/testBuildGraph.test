#Testing building the graph

# build the graph
LoadGraph marvel marvel.csv