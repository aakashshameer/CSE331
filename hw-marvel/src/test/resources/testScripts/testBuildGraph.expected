#Testing building the graph

# build the graph
loaded graph marvel